hack/triage.py

```diff
@@ -1,119 +0,0 @@
-import argparse
```

Comment:
This was a new traige script to hold us over until we had a UI, which we now do.